lib/menu.rb

```diff
@@ -0,0 +1,19 @@
+class Menu
+
+  attr_reader :nested_menu
+
+  def initialize
+    @nested_menu = [
+      ["chocolate", 0.50],
+      ["rasperries", 0.25],
+      ["ice_cream", 3.72]
+    ]
```

Comment:
Well named again, and thank you for using multi line to define menu to increase clarity.